### PR TITLE
Fix update capacity script

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

The update capacity script is currently not working.

## Description

Fixes the update capacity script and updates the Bulgaria capacities.

### Double check

- [x] I have tested my parser changes locally with `poetry run python scripts/ENTSOE_capacity_update.py  "zone_key"`
- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
Looks like the script nuked the emission factors, that's not great... 😅

I'll update the script to prevent this in the future.

**Comment by VIKTORVAV99** [comment]
Update on this: there seems to be something in https://github.com/electricitymaps/electricitymaps-contrib/blob/579cf5065fb308fef1379a5b9c1911d4927344fb/electricitymap/contrib/config/__init__.py that mutates the ZONE_CONFIG (running print right after the ZONE_CONFIG declaration and at the end of the file returns different results). I don't think that should be happening and will be looking into a fix.

In the meantime I'll open a new PR to fix the capacity for BG.

**Comment by VIKTORVAV99** [comment]
Quick fix to get this working as expected again but we should investigate why the ZONES_CONFIG was mutated in the first place and perhaps migrate other files that use it to use the reading function directly like I did in here.

**Comment by madsnedergaard** [comment]
Would you mind sharing how you run the script? I'm getting a "No module named 'xmltodict'" when trying 😅 
We could also add a small README inside `/scripts` to explain how to run the different scripts.

File: scripts/ENTSOE_capacity_update.py [hunk 1]
```diff
@@ -22,13 +22,17 @@
 import yaml
 from utils import ROOT_PATH, run_shell_command
 
-from electricitymap.contrib.config import CONFIG_DIR, ZONES_CONFIG
+from electricitymap.contrib.config import CONFIG_DIR
+from electricitymap.contrib.config.reading import read_zones_config
```
Comment (pierresegonne) [inline scripts/ENTSOE_capacity_update.py:26]: Is that function new?

Comment (VIKTORVAV99) [reply scripts/ENTSOE_capacity_update.py:26]: Yes and no, it was refractored out of the __init__.py in #5269 but it should be the same. It looks like something is not behaving as expected though but I don't know if that is a new behavior or if it just became evident now.

I'll try and look into this the coming weekend.

**Review by madsnedergaard — COMMENTED** [review]
When I run the script it also changes indentation for all other fields (contributors, etc.), but might be easier to fix after #5368

**Review by madsnedergaard — APPROVED** [review]

**Comment by VIKTORVAV99** [comment]
> Would you mind sharing how you run the script? I'm getting a "No module named 'xmltodict'" when trying 😅 We could also add a small README inside `/scripts` to explain how to run the different scripts.

I'll merge this as is now but I'll try and create some documentation for the scripts in there the coming days.

**Pull request merged**